frontend/src/views/CellGuide/components/common/OntologyDagView/style.ts

```diff
@@ -73,10 +74,16 @@ export const WarningTooltipIcon = styled(Icon)`
 
 interface StyledSVGProps {
   isDragging: boolean;
+  transformMatrix: TransformMatrix;
 }
 export const StyledSVG = styled.svg<StyledSVGProps>`
   cursor: ${({ isDragging }) => (isDragging ? "grabbing" : "grab")};
   touch-action: none;
+  g {
```

Comment:
manually control the translation and scale by overriding style applied by visx Zoom component.